cookbooks/clearwater/recipes/ellis.rb

```diff
@@ -45,6 +45,12 @@
 end
 
 # Create number pools (first normal, then PSTN)
+execute "infra_script" do
+  command "service clearwater-infrastructure restart"
+  user "root"
+  only_if { ::File.exists?('/etc/clearwater/shared_config') }
```

Comment:
I think this should trigger the `create_X` actions rather than relying on the order of the actions.